controllers/raycluster_controller.go

```diff
@@ -290,7 +325,7 @@ func (r *RayClusterReconciler) createOrUpdateOwnedResource(ctx context.Context,
 		return r.Create(ctx, controlled)
 	}
 
-	patchResult, err := PatchMaker.Calculate(found, controlled, patch.IgnoreStatusFields())
+	patchResult, err := PatchMaker.Calculate(found, controlled, PatchCalculateOpts...)
```

Comment:
does this need to get migrated to spark?

Reply:
Based on what we're doing now, this is not strictly necessary. But, yes, to make them both resilient to future changes, spark and ray should use these common opts.